bindings/python/setup.py

```diff
@@ -41,8 +53,5 @@ def get_py_version_cfgs():
     ],
     include_package_data=True,
     zip_safe=False,
-    install_requires=[
-        '<pyhumps> >= <3.8.0>',
-        '<dacite>'
-    ]
+    install_requires=load_requirements("requirements-dev.txt")
```

Comment:
This doesn't work for pypi? At least in the released packages I don't see requirements-dev.txt included